pkg/policy/distillery_test.go

```diff
@@ -1663,72 +1669,83 @@ func Test_EnsureDeniesPrecedeAllows(t *testing.T) {
 	identity := identity.NewIdentityFromLabelArray(identity.NumericIdentity(identityFoo), labelsFoo)
 
 	tests := []struct {
-		test   string
-		rules  api.Rules
-		result MapState
+		test     string
+		rules    api.Rules
+		expected MapState
 	}{
 		{"deny_world_no_labels", api.Rules{ruleL3DenyWorld, ruleL3AllowWorldIP}, newMapState(map[Key]MapStateEntry{
-			mapKeyL3WorldIngress: mapEntryDeny,
-			mapKeyL3WorldEgress:  mapEntryDeny,
+			mapKeyL3WorldIngress:  mapEntryDeny,
```

Comment:
Why are there now 2 keys for the same direction? Shouldn't a single wildcard be sufficient?

Reply:
Other CIDR identities are automatically selected by the reserved:World identity by the policy repository. This test was silently failing before, nothing in the port range work caused this change. We can and should address/understand these redundancies, but not in this PR.